_content/history.md

```diff
@@ -1,43 +1,14 @@
 ---
-layout: page
+layout: history
 title: History
 group: top
 current: history
 ---
 
-# History of The Nottingham New Theatre
+# What is The Nottingham New Theatre? 
```

Comment:
I'm not sure a title is needed here - it takes up a lot of space. Going straight in with "The nottingham new theatre is..." is pretty self explanatory.